index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Microfolio</title>
+    <link
+      rel="shortcut icon"
+      href="https://img.icons8.com/external-others-inmotus-design/67/null/external-B-alphabet-others-inmotus-design-22.png"
+      type="image/x-icon"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <nav>
+      <img
+        id="menu"
+        src="https://img.icons8.com/external-tal-revivo-regular-tal-revivo/96/null/external-mobile-application-hamburger-menu-setting-interface-basic-regular-tal-revivo.png"
+      />
+    </nav>
```

Comment:
- [ ] Great job adding the hamburger menu on the right side of the design the previous reviewer suggested. However, it currently looks bigger than the Figma design. I will suggest that you reduce the size of the hamburger menu so it looks similar to the Figma design :100: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/227284671-8ab0beef-3a99-4872-bf75-ec9e3a6dc398.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/227284504-ef800079-a62e-4b3b-8bd1-595503f3db9d.png)

Reply:
Fixed Thank you